typescript/cli/src/config/warp.ts

```diff
@@ -144,9 +144,20 @@ export async function createWarpRouteDeployConfig({
       'hyperlane-registry',
     );
 
-    const interchainSecurityModule = advanced
-      ? await createAdvancedIsmConfig(context)
-      : createDefaultWarpIsmConfig(owner);
+    let createDefaultIsm: boolean;
+    if (advanced) {
+      createDefaultIsm = false;
+    } else if (context.skipConfirmation) {
+      createDefaultIsm = true;
+    } else {
+      createDefaultIsm = await confirm({
+        message: 'Use a trusted ISM for warp route',
+      });
+    }
+
+    const interchainSecurityModule = createDefaultIsm
```

Comment:
So right now, if the user selects 'no' for "Use a trusted ISM for warp route", then they will be prompted to go into "advanced mode" and have them configure an ISM. We want them to instead go through these paths:

1. if --advanced is specified: manually configure ISM. in other words, call `createAdvancedIsmConfig()`
2. if --advanced is not specified, prompt "Do you want to use a trusted ISM for warp route?"
  - if 'yes' , create _both_ trustedRelayer and defaultFallbackRouting. See `createDefaultWarpIsmConfig()`
  - If 'no' , only create a defaultFallbackRouting.